vendor/bat-native-ads/src/bat/ads/internal/notifications.cc

```diff
@@ -49,76 +49,91 @@ void Notifications::Initialize(InitializeCallback callback) {
 bool Notifications::Get(const std::string& id, NotificationInfo* info) const {
   DCHECK(is_initialized_);
 
-  auto notification = notifications_.find(id);
-  if (notification == notifications_.end()) {
+  auto iter = std::find_if(notifications_.begin(), notifications_.end(),
```

Comment:
nit: please use `it` instead of `iter` for coding style consistency for ads

Reply:
It seems that google developers like name `iter` too :
https://google.github.io/styleguide/cppguide.html#Function_Comments

Reply:
@gdregalo Google use both `iter` and `it`, nit was for consistency